pixeltable/func/function.py

```diff
@@ -32,7 +33,7 @@ class Function(abc.ABC):
     is_method: bool
     is_property: bool
     _conditional_return_type: Optional[Callable[..., ts.ColumnType]]
-    __resolutions: list[Function]  # Cached overload resolutions of this function
+    _resolutions: list[Self]  # Cached overload resolutions of this function
```

Comment:
I don't think Self is available in python 3.9.

Reply:
Yes it is, via the `typing_extensions` package, which is how I'm importing it here.

I do all my development on 3.9 (and we have CI matrix configs that run against 3.9) so rest assured I've thought of and tested this :)
<!-- Reviewable comment -OEWkFnqCxx5qunbAz3n:b-xwh7h3 -->
<!-- Sent from Reviewable.io -->